js/enquiry.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+	if($('#enquiry-form').show()){
+		$('.fakeUpload').on('click',function (){
+			$('#file').click();
+		});
+		$('#file').on('change',function(){
+			var filename = this.value;
+		    var lastIndex = filename.lastIndexOf("\\");
+		    if (lastIndex >= 0)
+		        filename = filename.substring(lastIndex + 1);
+	    	$('#filename').html(filename);
+		});
+		 $('.submitForm').on('click', function(){
+		 	var checkedAtLeastOne = false;
+			$('input[type="checkbox"]').each(function() {
+			    if ($(this).is(":checked")) 
+			        checkedAtLeastOne = true;
+			});
+			if(!checkedAtLeastOne)
+				$('#checkbox1').prop("checked",true);
+			$('#submit').click();
```

Comment:
The submit function from jQuery or Javascript will skipped the default validation of the form.( It can send a form which haven't fill in all 'required' field)

If i want to use the submit function, i will need to do all the validation by jquery.